### DISCUSSION TITLE
Unit 6 is live!

### DISCUSSION THREAD

**Discussion by rasbt** [Lightning-AI/dl-fundamentals#33] [Announcements]
Excited to share that a new Deep Learning Fundamentals unit just launched!  🚀🎉
In this Unit 6, I am covering some of my favorite deep learning tips & tricks using the Lightning Trainer:

- learning rate schedulers & cosine annealing
- CLI configs for hparam sweeps
- debugging w. batch overfitting

https://lightning.ai/pages/courses/deep-learning-fundamentals/unit-6-overview-essential-deep-learning-tips-tricks/

Btw, we also have Discord channel now 🙌
https://discord.com/invite/XncpTy7DSt
